src/components/task/create-new/index.jsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import dayjs from "dayjs";
+import { useForm } from "react-hook-form";
+import { yupResolver } from "@hookform/resolvers/yup";
+import propTypes from "prop-types";
+
+import Button from "../../base/button";
+import Image from "../../base/image";
+import TextArea from "../../base/text-area";
+import Styles from "./index.module.scss";
+import { addNewTaskSchema } from "../../../utils/schema";
+import { useDispatch } from "react-redux";
+import { addTaskToTodo } from "../../../store/actions/todo";
+
+function CreateTask({ onSuccessfullTaskEntry }) {
+  const dispatch = useDispatch();
+
+  const addNewTask = (data) => {
```

Comment:
can we name it task?
```suggestion
  const addNewTask = (task) => {
```